pontoon/base/migrations/0018_auto_20150820_0925.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def change_chinese_plurals(apps, schema_editor):
+    Locale = apps.get_model('base', 'Locale')
+    Translation = apps.get_model('base', 'Translation')
+
+    for locale in Locale.objects.filter(name="Chinese"):
+        locale.nplurals = 1
+        locale.plural_rule = '0'
+        locale.cldr_plurals = '5'
```

Comment:
I don't know much about CLDR, so I'm assuming this is fine, but I'm curious: why this is set to 5?